src/Tribe/Rewrite.php

```diff
@@ -108,8 +117,9 @@ public static function instance() {
 	 *
 	 * @param WP_Rewrite|null $wp_rewrite
 	 */
-	public function __construct( WP_Rewrite $wp_rewrite = null ) {
+	public function __construct( WP_Rewrite $wp_rewrite = null, Translations_Loader $locale_switcher = null ) {
 		$this->rewrite = $wp_rewrite;
+		$this->translations_loader = tribe( Translations_Loader::class );
```

Comment:
```suggestion
		$this->translations_loader = $locale_switcher;
```
Was this the intended logic here...?

Reply:
The idea was to fallback on the instance provided by the Service Locator, but I forgot to take into account the input parameter; good catch, thanks.